src/discussion/srs-evaluation.tex

```diff
@@ -0,0 +1,55 @@
+% textidote: ignore begin
+\section{Software requirements evaluation}\label{sec:software-requirements-evaluation}
+% textidote: ignore end
+
+This section refers back to Section~\ref{sec:software-requirement-specification} and more specifically
+Table~\ref{tab:srs}.
+
+% textidote: ignore begin
+\subsection{Must haves}\label{subsec:must-haves}
+% textidote: ignore begin
+
+This section concerns the evaluation of the must-have requirements.
+Most must-have requirements are satisfied in the development of the first release of the ChessTeacher project.
+This entails the basic chess functionality, the online multiplayer functionality and the chess engine implementation.
+They work seamlessly and efficiently and are well integrated into the overall project.
+
+% textidote: ignore begin
+\subsubsection{Applied learning principles}
+% textidote: ignore begin
+
+Though most features are well implemented as mentioned above, there are some missing features.
+Regarding the applied learning principles the project is missing some key features regarding the learning principles
+mentioned in Section~\ref{sec:learning-methods}.
+Key missing features include explanations and interactivity.
```

Comment:
I understand the thought process of introducing the subsection, then the context and then the specific feature, but it makes the sentence read badly.

Reply:
Agree. Will fix